src/util/windows/version.rc

```diff
@@ -0,0 +1,49 @@
+//
+// Include the necessary resources
+//
+#include <winver.h>
+#include <ntdef.h>
+
+#ifdef RC_INVOKED
+
+//
+// Set up debug information
+//
+#if DEBUG
+#define VER_DEBUG VS_FF_DEBUG
+#else
+#define VER_DEBUG 0
+#endif
+
+// ------- version info -------------------------------------------------------
+
+VS_VERSION_INFO VERSIONINFO
+FILEVERSION             1,7,5,0
+PRODUCTVERSION          1,7,5,0
+FILEFLAGSMASK           VS_FFI_FILEFLAGSMASK
+FILEFLAGS               (VER_DEBUG|VS_FF_PRERELEASE)
+FILEOS                  VOS_NT
+FILETYPE                VFT_APP
+BEGIN
+	BLOCK "StringFileInfo"
+	BEGIN
+		BLOCK "040904b0"
+		BEGIN
+			VALUE "Comments",         "https://github.com/LinusDierheimer/fastfetch"
+			VALUE "CompanyName",      "Github"
+			VALUE "FileDescription",  "Like neofetch, but much faster because written in C."
+			//TODO: Generate automatically with git tag
+			VALUE "FileVersion",      "V1.7.5.0"
+			VALUE "InternalName",     "fastfetch.exe"
+			VALUE "LegalCopyright",   "MIT license"
+			VALUE "OriginalFilename", "fastfetch.exe"
+			VALUE "ProductName",      "fastfetch"
+			VALUE "ProductVersion",	  "V1.7.5.0"
```

Comment:
Can we configure this (especially the version) from cmake using `configure_file`?

Reply:
I have no idea how to do that.